spec/features/users/user_profile_spec.rb

```diff
@@ -115,11 +115,10 @@
         click_link 'Edit', href: manage_password_path
       end
 
-      expect(page).to_not have_css('#pw-strength-cntnr.display-none')
-      expect(page).to have_content '...'
+      expect(page).not_to have_content(t('instructions.password.strength.intro'))
 
       fill_in t('forms.passwords.edit.labels.password'), with: 'this is a great sentence'
-      expect(page).to have_content 'Great'
+      expect(page).to have_content t('instructions.password.strength.v')
```

Comment:
As a safeguard to ensure that the `not_to` above doesn't give us false confidence in case we remove that text at some point in the future, I'd suggest also including the positive assertion here.

```suggestion
      expect(page).to have_content(t('instructions.password.strength.intro'))
      expect(page).to have_content t('instructions.password.strength.v')
```